web/styles/typeahead.css

```diff
@@ -136,6 +136,9 @@
         /* Present as flexbox to better control
            icon alignment and spacing, when icons
            are shown. */
+        overflow: hidden;
+        max-width: calc(25ch);
```

Comment:
We don't want to hardcode 25 character limit here. Also this code changes should be above the comment.